bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -289,7 +290,13 @@ <h2>{{ _('You deserve peace of mind everywhere.') }}</h2>
         </div>
 
         <div class="show-fxa-supported-signed-out">
-          {{ fxa_email_form(entrypoint='accounts-page', utm_source='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': 'accounts-page', 'medium': 'referral'}, button_class='mzp-c-button mzp-t-primary mzp-t-product', button_text=_('Create an Account')) }}
+          {{ fxa_email_form(
+            entrypoint='mozilla.org-fxa_benefits',
```

Comment:
Is there a reason that this `entrypoint` is different to the one above, if it's coming from the same page?